### DISCUSSION TITLE
[solved] option to skip assets download

### DISCUSSION THREAD

**Discussion by RememberTheAir** [DominikPieper/obsidian-ReadItLater#72] [Ideas]
Hello there,

First off, thanks a lot for the great plugin. It's one of my favorites and it helps me a lot to backup webpages and make their content easily accessible in my personal vault I use to keep track of interesting stuffs and notes.

The only thing I would like to suggest is an option to skip assets download when we save an article. This would especially be useful when syncing a vault to a cloud service with limited storage or to a git repository, without having to keep a .gitignore entry updated

**Comment by adamluckdev** [comment]
Hi, 

there's option in plugin settings to disable downloading of images.

![Screenshot from 2023-01-12 10-56-13](https://user-images.githubusercontent.com/10731521/212036039-29126356-7547-4a83-96ce-b3ac7c9efa93.png)

**Reply by RememberTheAir** [reply]
Oh my god how did I miss that! Thank you :)
